### PR TITLE
Feature/d1.4.4 h3 profile ui changes

### PR DESCRIPTION
**Pull request by ahamblyn**
In the UI, users can now create an H3 profile and apply profile overrides to the target instance.

### CODE REVIEW

File: wct-core/src/main/webapp/jsp/target-profile.jsp [hunk 1]
```diff
@@ -42,14 +42,12 @@
   }
 
   function toggleProvideOverrides(harvesterTypeValueSelected) {
-    if (harvesterTypeValueSelected == "HERITRIX1") {
-      $('#excludeFiltersRow').show();
-      $('#forceAcceptFiltersRow').show();
-      $('#excludedMimeTypesRow').show();
+    if (harvesterTypeValueSelected == 'HERITRIX1') {
```
Comment (obrienben) [inline wct-core/src/main/webapp/jsp/target-profile.jsp]: Should statements like this be matching on the enum instead?

File: wct-core/src/test/java/org/webcurator/core/profiles/Heritrix3ProfileTest.java [hunk 2]
```diff
@@ -408,4 +414,94 @@ public final void testToProfileXmlHeritrix3Profile() {
...
+        // No overrides set to true
+        long modifiedDocumentLimit = 25;
+        double modifiedDataLimit = 100.0d;
+        double modifiedTimeLimit = 250.0d;
+        long modifiedMaxPathDepth = 150;
+        long modifiedMaxHops = 50;
+        long modifiedMaxTransitiveHops = 5;
+        String modifiedIgnoreRobots = "ignore";
+        boolean modifiedIgnoreCookies = true;
+        List<String> modifiedBlockUrls = new ArrayList<String>();
```
Comment (obrienben) [inline wct-core/src/test/java/org/webcurator/core/profiles/Heritrix3ProfileTest.java]: Barely worth mentioning, but the ProfileOverrides class implements the URL lists as Linked Lists, and here they're Array Lists.

File: wct-core/src/main/java/org/webcurator/core/profiles/PolitenessOptions.java [hunk 3]
```diff
@@ -0,0 +1,127 @@
+package org.webcurator.core.profiles;
+
+public class PolitenessOptions {
+    private double delayFactor;
+    private long minDelayMs;
+    private long maxDelayMs;
+    private long respectCrawlDelayUpToSeconds;
+    private long maxPerHostBandwidthUsageKbSec;
+    public static final PolitenessOptions POLITE_OPTIONS = new PolitenessOptions(10.0d, 9000, 90000, 900, 400);
+    public static final PolitenessOptions MEDIUM_OPTIONS = new PolitenessOptions(5.0d, 3000, 30000, 300, 800);
```
Comment (obrienben) [inline wct-core/src/main/java/org/webcurator/core/profiles/PolitenessOptions.java]: @ahamblyn Could these politeness defaults be configurable via spring? Just trying to think of a way they could be updated in the future without requiring a code change.

File: wct-core/src/main/webapp/jsp/target-profile.jsp (hunk 1 above)
Comment (ahamblyn) [reply wct-core/src/main/webapp/jsp/target-profile.jsp]: This is Javascript

**Pull request merged**